jsx/FilterForm.js

```diff
@@ -162,7 +162,11 @@ class FilterForm extends React.Component {
     }
 
     return (
-      <Panel id="selection-filter" title="Selection Filter">
+      <Panel
+        id={this.props.id}
+        height={this.props.height}
+        title={this.props.title}
```

Comment:
But won't the warning point to the inner component? It's useful to have the type checking at both levels.
It also acts as documentation. As a client of `FilterForm` I can see what I can pass as a prop without digging into the implementation.